packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -171,7 +171,7 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     if (
       newProps.checkboxVisibility !== checkboxVisibility ||
       newProps.columns !== columns ||
-      newProps.viewport.width !== viewport.width
+      newProps.viewport!.width !== viewport!.width
```

Comment:
Yeah, generally good to avoid using it, but super helpful to get the codebase in a state where strictNullChecks can be enabled